src/pluginenums.js

```diff
@@ -10,6 +10,7 @@ export default {
     ERROR: 'error'
   },
   ERROR_CODES: {
+    MANIFEST_PARSE: 7,
     BUFFERING_TIMEOUT: 8,
     MANIFEST: 9
```

Comment:
Could we change this to MANIFEST_LOAD?

Reply:
Sorry, maybe GitHub formatting, I mean the one on line 15, that's currently just MANIFEST.